src/framework/vst/vsttypes.h

```diff
@@ -63,14 +63,23 @@ enum class VstPluginType {
     Fx
 };
 
-struct VstPluginMeta {
-    PluginId id;
-    VstPluginType type = VstPluginType::Undefined;
-    std::string name;
-    std::string path;
-};
-
-using VstPluginMetaList = std::vector<VstPluginMeta>;
+/// @see https://steinbergmedia.github.io/vst3_doc/vstinterfaces/namespaceSteinberg_1_1Vst_1_1PlugType.html
+namespace PluginCategory {
+static const std::string Analyzer = "Analyzer";
```

Comment:
Can we use constexpr std::string_view here?

Reply:
it's not gonna be copied, but I'm not against
fixed